.github/workflows/ci_frameworks.yml

```diff
@@ -0,0 +1,67 @@
+name: ci_frameworks
+
+on:
+  push:
+    paths:
+      - "mesonbuild/dependencies/**"
+      - "test cases/frameworks/**"
+      - ".github/workflows/frameworks.yml"
+  pull_request:
+    paths:
+      - "mesonbuild/dependencies/**"
+      - "test cases/frameworks/**"
+      - ".github/workflows/frameworks.yml"
+
+jobs:
+
+  scalapack_linux:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - run: python -m pip install .
+    - name: install prereq
+      run: |
+        sudo apt update -yq
+        sudo apt install -yq --no-install-recommends pkg-config ninja-build gfortran liblapack-dev libscalapack-mpi-dev libopenmpi-dev openmpi-bin
```

Comment:
This needs to be in the regular travis CI image. All Linux dep tests must be in the same place. Does it require any other additional packages than `libscalapack-mpi-dev`?

Reply:
OK--right only thing new is `liblapack-dev libscalapack-mpi-dev`

Reply:
so those would have to be added to Docker or?

Reply:
For example, HDF5 is in Docker, but I added a GitHub Action to do MacOS as well. When MSYS2 is ready for GitHub Action, I could add that for HDF5 also.

For Scalapack, if I'm understanding correctly, we would add Scalapack to Docker and then perhaps have this GitHub Action for Scalapack on MacOS? 

For many other prereqs, we can also do Windows--OpenMPI is generally not available for Windows so I didn't make an Action for Scalapack on Windows.

Reply:
Due to access permissions I have to update the image manually. I have started the image build, I'll post an update once it has finished uploading so you can restart the build.

However do update the `Dockerfile` to have the new packages.

Reply:
Docker image updated.